content/quickstart.md

```diff
@@ -7,29 +8,72 @@ tocopen: true
 
 # Prerequisites
 
-Under construction.
+1- OpenSeeFace tracking: webcam
 
-# Installation
+2- IFacialMocap tracking: iOS device with face ID or A12 (or newer) chip

```

Comment:
> face ID or A12 (or newer) chip

This is redundant. Would be better to specify a specific model of Apple device, in this case the iPhone X